src/index.html

```diff
@@ -8,7 +8,48 @@
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home" class="logoLink">
+        <img
+          src="/images/logo.png"
+          alt="logo"
```

Comment:
alt attribute should be descriptive, checkout checklist point 5
```suggestion
          alt="Moyo logo"
```